ops/terraform/services/server/server-regression/main.tf

```
@@ -155,3 +157,58 @@ resource "aws_lambda_function" "glue_trigger" {
 
   role = aws_iam_role.glue_trigger.arn
 }
+
+resource "aws_lambda_function" "spice_refresh_trigger" {
+  description      = "Triggers a full SPICE refresh when the server regression glue crawler completes"
+  architectures    = ["x86_64"]
+  function_name    = "bfd-${local.env}-${local.service}-refresh-spice-trigger"
+  handler          = "spice-trigger.lambda_handler"
+  memory_size      = 128
+  package_type     = "Zip"
+  role             = aws_iam_role.spice_refresh_role.arn
+  runtime          = "python3.9"
+  filename         = data.archive_file.spice_trigger.output_path
+  source_code_hash = data.archive_file.spice_trigger.output_base64sha256
+  timeout          = 3
```

```suggestion
  timeout          = 60
```

The `timeout` is the number of seconds before the Lambda is forced to exit by the Runtime. I suspect 3 seconds is _probably_ enough for this Lambda in _most_ cases, but it doesn't hurt to raise it a bit to give it more wiggle room.